client/coral-admin/src/components/Comment.js

```diff
@@ -30,7 +31,18 @@ export default props => {
           {links ?
           <span className={styles.hasLinks}><Icon name='error_outline'/> Contains Link</span> : null}
           <div className={`actions ${styles.actions}`}>
-            {props.modActions.map((action, i) => getActionButton(action, i, props))}
+            {props.modActions.map(
+              (action, i) =>
+              <ActionButton
```

Comment:
Yeah, it's getting used in two places so it makes sense to make it its own component.